mne/viz/_brain/_brain.py

```diff
@@ -360,12 +364,24 @@ class Brain(object):
        +---------------------------+--------------+---------------+
        | labels                    | ✓            | ✓             |
        +---------------------------+--------------+---------------+
+       | remove_data               |              | ✓             |
+       +---------------------------+--------------+---------------+
        | remove_foci               | ✓            |               |
        +---------------------------+--------------+---------------+
+       | remove_head               |              | ✓             |
+       +---------------------------+--------------+---------------+
        | remove_labels             | ✓            | ✓             |
        +---------------------------+--------------+---------------+
        | remove_annotations        | -            | ✓             |
        +---------------------------+--------------+---------------+
+       | remove_sensors            |              | ✓             |
+       +---------------------------+--------------+---------------+
+       | remove_skull              |              | ✓             |
+       +---------------------------+--------------+---------------+
+       | remove_text               |              | ✓             |
+       +---------------------------+--------------+---------------+
+       | remove_volume_labels      |              | ✓             |
+       +---------------------------+--------------+---------------+
```

Comment:
Rather than a bunch of `remove_*` methods, an alternative would be to make all `add_*` functions return actors, and have a single `remove_actors` method that took an actor or list of actors. I think I like that design slightly better because it also allows for configuring actor VTK properties if people want to do so.

Reply:
We could make `actors` a public property. One difficulty is that then for sensors you'd have to return a dictionary with all the types if you want to support removing by type (e.g. `fnirs`), that could work also, but it puts a bit more work on the user

Reply:
I think I prefer +0.5 a single `.remove` method if it would work. @GuillaumeFavelier @agramfort any opinion on having `remove_head(), remove_volume_labels(), ...` vs `remove_actors(actors)`?

Reply:
... if we really want, we could add convenience aliases like `remove_actors('head')` or so, but I don't think we'd need it

Reply:
The only thing is that there are already `remove_annotation` and `remove_labels` so as it is would be consistent stylistically...

I'll push another commit the other way and then we can revert if the individual methods are better

Reply:
Ahh you're right. I think it makes sense then to leave `remove_*` rather than a generic `remove_actors`

Reply:
Also in `add_text` someone spelled out the way to do it with adding a name etc. that would be an internal reference to the actor which would suggest doing it the way it is now, otherwise the name of the text is extraneous, that's something the user should keep track of

Reply:
And in the case of `add_annotation()` and `add_label()` they do not rely on `add_actor()` but `add_overlay()` instead.

Reply:
They follow the `_LayeredMesh` logic instead of the usual `vtkActor` to avoid geometry duplication and opacity issues since VTK9.